cmake/FindFFTW3.cmake

```diff
@@ -75,17 +75,16 @@ set(_check_list)
 if (WIN32)
 
     foreach(_lib ${_libraries})
-
       string(TOUPPER ${_lib} _LIB)
 
-      find_library(${_LIB}_LIBRARY NAMES lib${_lib}-3 lib${_lib}
+      find_library(${_LIB}_LIBRARY lib${_lib}-3 ${_lib}
         HINTS $ENV{FFTW3_ROOT_DIR} PATH_SUFFIXES lib)
       mark_as_advanced(${_LIB}_LIBRARY)
       list(APPEND FFTW3_LIBRARIES ${${_LIB}_LIBRARY})
       list(APPEND _check_list ${_LIB}_LIBRARY)
     endforeach()
 
-    message(STATUS "FFTW3 WINDOWS libraries: " ${FFTW3_LIBRARIES})
+    message("FFTW3 WINDOWS libraries: " ${FFTW3_LIBRARIES})
```

Comment:
just checking, why do you remove the `STATUS`? From the CMake doc:
```
(none) or NOTICE
Important message printed to stderr to attract user’s attention.

STATUS
The main interesting messages that project users might be interested in. Ideally these should be concise, no more than a single line, but still informative.
```

Reply:
Yeah, no reason really. I was coping a section from the Gadgetron repo that does this a bit differently. I put it back in.